app/javascript/article-form/components/ImageUploader.jsx

```diff
@@ -96,11 +97,18 @@ export const ImageUploader = () => {
       'image-markdown-copy-link-input',
     );
 
-    Runtime.copyToClipboard(imageMarkdownInput.value).then(() => {
-      dispatch({
-        type: 'show_copied_image_message',
+    Runtime.copyToClipboard(imageMarkdownInput.value)
+      .then(() => {
+        dispatch({
+          type: 'show_copied_image_message',
+        });
+      })
+      .catch((error) => {
+        addSnackbarItem({
+          message: error,
+          addCloseButton: true,
+        });
```

Comment:
The `Runtime.copyToClipboard` was rejecting the Promise and we didn't have this `catch` clause before. This adds context to the user when encountering an error.